src/StructuredLogger/StructuredLogger.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>

```

Comment:
Curious why keeping this netstandard2.0 wasn't enough?

Reply:
To avoid warnings during nuget restore, since a few version MsBuild is not targetting netstandard2.0 anymore. I also have other pending changes where I update some NuGet references, I'll shoot another PR.

Reply:
I'm speaking about thoses warnings:
```
warning NU1701: Package 'DeltaCompressionDotNet 1.0.0' was restored using '.NETFramework,Version=v4.6.1, .NETFramework,Version=v4.6.2, .NETFramework,Version=v4.7, .NETFramework,Version=v4.7.1, .NETFramework,Version=v4.7.2, .NETFramework,Version=v4.8' instead of the project target framework '.NETCoreApp,Version=v3.1'. This package may not be fully compatible with your project.
```

The most problematic is the Squirrel package.

Reply:
I guess a lot of users rely on just downloading StructuredLogger.dll and using it in their builds and it's convenient for them to have a single one in all cases.

Your fix makes sense, but I'll need to think about it more whether I'm ready to jump forward with targeting. Afraid to break consumers.

As a workaround you can add NoWarn to shut up NU1701.